### DISCUSSION TITLE
Setting up a service on void linux

### DISCUSSION THREAD

**Discussion by VoidAlone** [Spotifyd/spotifyd#1290] [Q&A] [answered]
There are instructions in the wiki for starting a service assuming that you're using systemd. 

In the case of void linux, which uses runit, what would be the best approach for setting up a service (user or system service)?

**Comment by eladyn** [comment, marked as answer]
Sorry for taking so long to answer. It's been a long since time since I last tried Void and I don't exactly remember how runit works, but starting `spotifyd` should be as easy as running `spotifyd --no-daemon` within the `run` file (the `--no-daemon` makes sure that `spotifyd` doesn't fork into background and is managed by the service manager).

**Reply by VoidAlone** [reply]
Hey thanks for your reply! This is exactly what I ended up doing. I'm running i3, and it's probably a bad place to put it, but I added `exec spotifyd --no-daemon` to my i3 config. Sorry for the potentially pointless question, I'm new to linux and learning a lot all at once, but hopefully this will be useful for another noob in the future.
